### PR TITLE
github markdown displayed in human-pane

### LINKED ISSUE #315: Simplify panes and display github markdown flavor in human-pane
The actual markdown-pane do not display the full markdown flavor.
A panes simplification can be achieved displaying markdown directly in human-pane

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

File: src/humanReadablePane.js [hunk 1]
```diff
@@ -86,6 +81,28 @@ module.exports = {
...
     }
 
+    //  @@ When we can, use CSP to turn off scripts within the iframe
+    div.setAttribute('class', 'docView')
+    const iframe = myDocument.createElement('IFRAME')
+    let dataUri
+
+    // render markdown to html
+    const markdownHtml = function () {
+      kb.fetcher.webOperation('GET', subject.uri).then(response => {
```
Comment (angelo-v) [inline src/humanReadablePane.js:91]: **question** why do we need another fetch here? The resource should already have been fetched, hasn't it?

Comment (bourgeoa) [reply src/humanReadablePane.js:91]: The original fetch is a HEAD only to find the contentType. We need the content now to feed the `data:uri`.

File: src/humanReadablePane.js [hunk 2]
```diff
@@ -86,6 +81,28 @@ module.exports = {
...
+        const res = marked.parse(response.responseText)
+        dataUri = 'data:text/html;charset=utf-8,' + encodeURIComponent(res)
+        iframe.setAttribute('src', dataUri) // allow-same-origin
```
Comment (angelo-v) [inline src/humanReadablePane.js:94]: **suggestion (non-blocking)** you could just render the markdown html into a div, no need for an iframe in this case

Comment (bourgeoa) [reply src/humanReadablePane.js:94]: I decided to keep the same html display for both. It is a lazy solution. The performance is quite good.

Comment (angelo-v) [reply src/humanReadablePane.js:94]: I am fine with that

**Review by angelo-v — APPROVED** [review]

**Pull request merged**